src/diskquota.c

```diff
@@ -381,7 +381,7 @@ disk_quota_worker_main(Datum main_arg)
 		int major = -1, minor = -1;
 		int has_error = worker_spi_get_extension_version(&major, &minor) != 0;
 
-		if (major == DISKQUOTA_MAJOR_VERSION && minor == DISKQUOTA_MINOR_VERSION) break;
+		if (major == DISKQUOTA_MAJOR_VERSION) break;
```

Comment:
I think we shouldn't do that!

Reply:
Suppose the user had version 2.1 installed and upgraded the library to version 2.2.
Then, at startup, diskquota will issue a warning and offer to update the extension.
But with this patch, the diskquota will continue to work with the old SQL part.

Reply:
If user will use our version of diskquota, diskqouta version 2.1 will be the same as diskquota version 2.2, because diskquota 2.1 is symlink of diskquota 2.2.

Reply:
Yes, but in the database the user has version 2.1 before the update and he does not know that he needs to update!